### PR TITLE
Javadoc fixes

### PR DESCRIPTION
**Pull request by pvliss**
Yet some more minor javadoc fixes(less this time :) ) as I did another pass comparing with the Kotlin code to make sure they are aligned.

-   Please make sure the below checklist is followed for Pull Requests.

-   [x] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
-   [x] Tests are added where necessary
-   [x] Documentation is added/updated where necessary
-   [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/server/templates/src/main/java/package/security/PersistentTokenRememberMeServices.java.ejs [hunk 1]
```diff
@@ -180,6 +180,10 @@ public class PersistentTokenRememberMeServices extends
      * <p>
      * The standard Spring Security implementations are too basic: they invalidate all tokens for the
      * current user, so when he logs out from one browser, all his other sessions are destroyed.
+     *
+     * @param request the request.
```
Comment (jdubois) [inline generators/server/templates/src/main/java/package/security/PersistentTokenRememberMeServices.java.ejs]: I'm not a big fan of "response the response", I'd rather have no Javadoc here, as it doesn't help at all

**Comment by jdubois** [comment]
Thanks a lot for the hard work @pvliss !
I made a small comment but I'll correct this myself

File: generators/server/templates/src/main/java/package/security/PersistentTokenRememberMeServices.java.ejs (hunk 1 above)
Comment (jdubois) [reply generators/server/templates/src/main/java/package/security/PersistentTokenRememberMeServices.java.ejs]: OK let's keep this for consistency with the other methods, as it's the only one in this case

Comment (pvliss) [reply generators/server/templates/src/main/java/package/security/PersistentTokenRememberMeServices.java.ejs]: @jdubois Actually me neither. Its mainly to keep the javadoc tool happy :)

**Comment by pvliss** [comment]
Thanks for merging @jdubois

**Pull request merged**